docs/user/data_management.rst

```diff
@@ -276,6 +276,123 @@ features), you must explicitly enable "mixed" indexing mode with ``geomesa.mixed
++-----------+---------+----------------------------------------------------------------------------+
+| Record    | 1       |                                                                            |
++-----------+---------+----------------------------------------------------------------------------+
+| Attribute | 1       |                                                                            |
++-----------+---------+----------------------------------------------------------------------------+
+
+Schema version 6 - 1.1.0-rc.3 through 1.2.0
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
++-----------+---------+----------------------------------------------------------------------------+
+| Index     | Version | Notes                                                                      |
++===========+=========+============================================================================+
+| Z3        | 1       |                                                                            |
++-----------+---------+----------------------------------------------------------------------------+
+| GeoHash   | 1       |                                                                            |
++-----------+---------+----------------------------------------------------------------------------+
+| Record    | 1       |                                                                            |
++-----------+---------+----------------------------------------------------------------------------+
+| Attribute | 2       | Added a composite date index and improved row-key collisions               |
++-----------+---------+----------------------------------------------------------------------------+
+
+
+Schema version 7 - 1.2.1
+^^^^^^^^^^^^^^^^^^^^^^^^
+
++-----------+---------+----------------------------------------------------------------------------+
+| Index     | Version | Notes                                                                      |
++===========+=========+============================================================================+
+| Z3        | 2       | Added support for non-point geometries and sharding for improved ingestion |
++-----------+---------+----------------------------------------------------------------------------+
+| GeoHash   | 1       |                                                                            |
++-----------+---------+----------------------------------------------------------------------------+
+| Record    | 1       |                                                                            |
++-----------+---------+----------------------------------------------------------------------------+
+| Attribute | 2       |                                                                            |
++-----------+---------+----------------------------------------------------------------------------+
+
+
+Schema version 8 - 1.2.2 through 1.2.4
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
```

Comment:
I put a note by Z2 that it replaces the geohash index - was it not clear though?